test/integration/logic/BUILD

```diff
@@ -64,7 +64,7 @@ host_compatible_java_test(
         "@graknlabs_graql//java:graql",
         "@graknlabs_common//:common",
     ],
-    data = [":basic-schema.gql"],
+    data = [":basic-schema.gql", ":test-type-resolution.gql"],
```

Comment:
I ad-hoc added another schema, but it's better to stick to one so I will fix this in the rewrite